lib/Doctrine/entities/ServiceType.php

```diff
@@ -38,8 +38,16 @@ class ServiceType {
     /** @OneToMany(targetEntity="Service", mappedBy="serviceType") */
     protected $services = null;
 
+    /**
+     * An instance of a Service of this ServiceType may
+     * unmonitored while in production.
+     * @Column(type="boolean", nullable=true)
```

Comment:
Do we definitely want to allow null as a value? Would it not be preferable to have a default?

Reply:
This is the reason for the draft pull, I'm revisiting this very line, trying to understand why I had a problem originally.

Reply:
Sorry - I hadn't spotted this was a draft

Reply:
A default of `false` does seem sensible.

Reply:
More ways to skin a cat -
`    public function __construct() {
        $this->services = new ArrayCollection();
        $this->allowMonitoringException = FALSE;
    }`
Which one could argue is more readable than the docblock annotation. But I won't, as the problem I had was the schema update tanking.